opentext-contact-us-page.html

```diff
@@ -0,0 +1,116 @@
+                </div>        
+                <form action="/action-page.php">
+                    <label for="inquiry">Inquiry type</label><br>
+                    <select>
+                        <option value="" disabled selected hidden>-- Please select --</option>
+                        <option>General inquiries</option>
+                        <option>Customer support</option>
+                        <option>Adoption & Learning Services</option>
+                        <option>Partners</option>
+                        <option>Professional Services</option>
+                        <option>Sales</option>
+                    </select>
+                    <label for="email">Email address</label> <br>
+                    <input type="text" id="email" name="Email"><br>
+                    <label for="fname">First name</label><br>
+                    <input type="text" id="fname" name="fname"><br>
+                    <label for="lname">Last name</label><br>
+                    <input type="text" id="lname" name="lname"><br>
+                    <label for="bphone">Business phone</label><br>
+                    <input type="text" id="bphone" name="bphone"><br>
+                    <label for="company">Company</label><br>
+                    <input type="text" id="company" name="company"><br>
+                    <label for="title">Title</label><br>
+                    <input type="text" id="title" name="title"><br>
+                    <label for="industry">Industry</label><br>
+                    <select>
+                        <option value="" disabled selected hidden>-- Please select --</option>
+                        <option>Administrative & Support Services</option>
+                        <option>Aerospace</option>
+                        <option>Agriculture, Forestry, Fishing & Hunting</option>
+                        <option>Automotive</option>
+                        <option>Banking</option>
+                        <option>Chemicals</option>
+                        <option>Consumer Goods</option>
+                        <option>Data Processing & Intelligence</option>
+                        <option>Defense & Intelligence</option>
+                        <option>Education</option>
+                    </select>
+                    <br>
+                    <label for="country">Country</label><br>
+                    <select>
+                        <option value="" disabled selected hidden>--Please select --</option>
+                        <option>Afghanistan</option>
+                        <option>Alalnd island</option>
+                        <option>Albania</option>
+                        <option>Algeria</option>
+                        <option>American Samoa</option>
+                    </select>
+                    <br>
+                    <p>How can we help?</p> 
+                    <textarea rows="4" cols="60" name="comment" form="usrform"></textarea><br>   
```

Comment:
Same thing with <br> here at the end of this line, instead of at the beginning of line 110